src/components/avatar/avatar.tsx

```diff
@@ -28,16 +28,16 @@ export class Avatar {
   @Prop({ reflect: true }) scale: Scale = "m";
 
   /** Specifies the `src` to an image (remember to add a token if the user is private). */
-  @Prop() thumbnail: string;
+  @Prop({ reflect: true }) thumbnail: string;
```

Comment:
@jcfranco would some of these be content? Like `thumbnail`, `fullName`? Maybe the "content" part is a little vague and we can make the rule more concrete by defining it. Would "content" be a sentence or sentences? Anything with spaces in it?

Reply:
I actually had doubts about these avatar props because of what you mentioned. I proceeded to reflect since I saw these more along the lines of `icon` where it does affect the UI, but not 1-1 (i.e., the icon name isn't rendered).